src/python/Dockerfile

```diff
@@ -1,240 +1,68 @@
-
-# build
-FROM grapl-python-build AS grapl-analyzerlib-build
-
-COPY --chown=grapl src/python/grapl_analyzerlib grapl_analyzerlib
-COPY --chown=grapl --from=graplinc-grapl-api-build /home/grapl/venv venv
-
-RUN --mount=type=bind,source=dist,target=/grapl-dist cd /grapl-dist && \
-    pip install grapl_common-*.whl
-
-RUN cd grapl_analyzerlib && \
-    pip install . && \
-    python setup.py sdist bdist_wheel
-
-# test
-FROM grapl-analyzerlib-build AS grapl-analyzerlib-test
-
-## Install test requirements
-COPY --chown=grapl --from=python-test-deps /home/grapl/python_test_deps python_test_deps
-RUN python_test_deps/install_requirements.sh
-
-# engagement-creator (this will die when we merge data-tenancy from main into nomad)
-################################################################################
-
-# build
-FROM grapl-python-build AS engagement-creator-build
-
-COPY --chown=grapl src/python/engagement-creator engagement-creator
-COPY --chown=grapl --from=grapl-analyzerlib-build /home/grapl/venv venv
-
-RUN cd engagement-creator && \
-    pip install .
-
-# deploy
-FROM grapl-python-deploy AS engagement-creator-deploy
-
-COPY --chown=grapl --from=engagement-creator-build /home/grapl/venv venv
-COPY --chown=grapl --from=engagement-creator-build /home/grapl/engagement-creator engagement-creator
-
-# Awful, stupid hacks around Python packaging.
-# But this will go away in like 2 days when I merge in data-tenancy.
-WORKDIR /home/grapl/engagement-creator
-RUN mv engagement_creator/run.py ./
-CMD python3 ./run.py
-
+ENTRYPOINT ["/bin/bash", "-o", "errexit", "-o", "nounset", "-c"]
 
 # analyzer-executor
 ################################################################################
+FROM grapl-python-base AS analyzer-executor
+COPY ./dist/analyzer-executor.pex /home/grapl/analyzer-executor.pex
+CMD ["./analyzer-executor.pex"]
 
-# build
-FROM grapl-python-build AS analyzer-executor-build
-
-COPY --chown=grapl src/python/analyzer_executor analyzer_executor
-COPY --chown=grapl --from=grapl-analyzerlib-build /home/grapl/venv venv
-
-RUN cd analyzer_executor && \
-    pip install .
-
-# deploy
-FROM grapl-python-deploy AS analyzer-executor-deploy
-
-COPY --chown=grapl --from=analyzer-executor-build /home/grapl/venv venv
-COPY --chown=grapl --from=analyzer-executor-build /home/grapl/analyzer_executor analyzer_executor
-
-CMD python3 analyzer_executor/src/run.py
-
-# test
-FROM analyzer-executor-build AS analyzer-executor-test
-
-ENV IS_RETRY=False
-
-## Install test requirements
-COPY --chown=grapl --from=python-test-deps /home/grapl/python_test_deps python_test_deps
-RUN python_test_deps/install_requirements.sh
-
-# model-plugin-deployer
+# engagement-creator
 ################################################################################
+FROM grapl-python-base AS engagement-creator
+COPY ./dist/engagement-creator.pex /home/grapl/engagement-creator.pex
+CMD ["./engagement-creator.pex"]
 
```

Comment:
actual-new-change-from-this-pr

basically copying the pattern from the above analyzer-executor, which is new as of jgrillo's data-tenancy thing